Load a folder of views

Mte90 opened the pull request:
This fix https://github.com/valberg/django-view-decorator/issues/13

In urls.py it is required that code:

```
import os
root = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
urlpatterns = [
path("", include_view_urls(extra_modules=[root + "/portal/views"])),
]
```

In this way loads from a folder the various views automatically.
I will update the readme tomorrow (also some parameters like login_required are not documented, I will do that).

Mte90 committed:
fix(urls): load a folder

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

Mte90 committed:
feat(readme): updated with the parameter list

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

Mte90 commented:
Now the readme is updated but require a review from @valberg

valberg commented:
Hi @Mte90 

Thanks for the pull request.

I'm not sure that this is the right approach. I would much rather that we traverse the contents of a python module than directly interacting with the filesystem. I think that could be accomplished by using `pkg_util` from the standard library.

Also I think we should be explicit about how the functionality is used. One should be explicit about wanting to scan all modules in a package, so we should introduce a new keyword argument "extra_packages" to solve this case.

Basically I want the API to be:

```
include_view_urls(
    extra_modules=["dotted.path.to.module"],
    extra_packages=["dotted.path.to.package"],
)
```

Mte90 committed:
feat(extra_packages): moved to new parameter

Mte90 commented on README.md:
```diff
@@ -61,6 +61,26 @@ class FooList(ListView):
+
+* `paths`: is in a plural form because support also a list of paths
+* `name` the classic route name
+* `namespace`: {I don't know what to write here}
```
it is required a review here

Mte90 commented:
So I added the new parameter and updated the readme but about `pkg_util` I don't know how to proceed.

To me is better if we don't know need to to define a variable with the root folder so it should be more fast butI don't know how to achieve it, I am not so much experience in this python knowledge.

Mte90 commented:
Ping @valberg

Mte90 commented:
ping @valberg

Mte90 commented:
ping @valberg

Mte90 commented:
Why you closed it?
I don't see that you implemented that feature

Mte90 commented:
didn't got any answer
